src/Microsoft.DotNet.VersionTools.net45/project.json

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "Newtonsoft.Json": "7.0.1",
+    "NuGet.Versioning": "3.5.0-beta2-1456",
+    "System.Collections.Immutable": "1.1.37",
```

Comment:
This project.json is binary equal to `Microsoft.DotNet.Build.Tasks.net45/project.json`, and I'll need to look at this file more closely to see what's needed. (This particular reference, probably not.)